src/actions/util.js

```diff
@@ -0,0 +1,3 @@
+export function action(type, data = {}) {
+  return {type, ...data};
+}
```

Comment:
All this is doing is helping to enforce the absolute requirement which is our reducers depend on the type key. Data is optional, type is not.
I, personally, would actually make this...
```js
return {type, data};
```
...so that all actions conform to a consistent api, although I appreciate this is potentially pedantic.